server/server.js

```diff
@@ -77,7 +78,8 @@ module.exports = async ({graphQLConfig, graphiqlConfig, postgresConfig, schemaLi
           await connectDataSources(newSchema.dataSources)
           schema = newSchema.schema
           dataSources = newSchema.dataSources
-          newSubScriptionServer(server, schema)
+          subscriptionServer.close()
+          subscriptionServer = newSubscriptionServer(server, schema)
```

Comment:
the only thing I can tell is maybe close the existing subscription server at line 70 and create a new one in 2 places:
1. just after line 78 (current one)
2. just after line 88 (error case when new schema is shit and we still need to start the subscription server with the old schema) 

sorry for being this pessimistic about exceptions :)

Reply:
Good catch